test/TestProjects/BodyAndPath-LowLevel/BodyAndPath-LowLevel.json

```diff
@@ -172,6 +172,54 @@
             }
           }
         }
+      },
+      "/{item3}/item4/{item2}/item1": {
```

Comment:
I'm a little confused by this test -- would it be possible add a description field to state what it is testing and what the expected outcome is?

For example, it looks like `item4` is listed as a query parameter, but given its position in the path, isn't it a path parameter?

Reply:
Every parameter has a description where it should go. I can add an explanation why it should go there.

Reply:
Done